manifests/grub.pp

```diff
@@ -0,0 +1,69 @@
+# === Copyright
+#
+# Copyright 2018, Kumina B.V., Tim Stoop
+# Licensed under the Apache License, Version 2.0 (the "License");
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# == Class: os_hardening::grub
+#
+# Hardens the grub config
+#
+class os_hardening::grub (
+  Boolean $enable                = false,
+  String  $user                  = 'root',
+  String  $password_hash         = false,
+  Boolean $boot_without_password = true,
+) {
+
+  case $::operatingsystem {
+    debian, ubuntu: {
+      $grub_cfg = '/boot/grub/grub.cfg'
```

Comment:
is it a better way to call `update-grub` on ubuntu/debian?

Reply:
I wouldn't bother:
```
~$ cat /usr/sbin/update-grub
#!/bin/sh
set -e
exec grub-mkconfig -o /boot/grub/grub.cfg "$@"
```
This is more consistent, imho.

Reply:
:D I did not check the file itself, but I saw it in all possible man pages :)
Fine for me as is now